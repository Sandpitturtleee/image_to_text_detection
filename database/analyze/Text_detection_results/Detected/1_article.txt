ziśpierwszeurodzinyte D go symbolukobicccj siły isiostrzenstwa ponadpodziałami -ubiegłego 3 października powstrzymałyśmy rządprzcd wprowadzeniem całkowitego zakazu aborcji. Alcwładzanicśpi-torządnocnych jarków -iby nie dać się zaskoczyć, nie możemy parasolek składac. Po pierwsze 'dlatego; żechcą nam zabraćin vitro. Po drugic: środowiska antykobiece; PiS sprzyja, wciąż zbicrają podpisy projcktami ustaw całkowicie zakazującymi aborcjiiantykonccpcji Równocześnie Ito po trzecie dobra ziana w szkole edukację scksualną zastapiłakatechizmem. Po czwarte -utrudnia do antykonccpcji Ta awaryjna którym pod dostęp blctka ,dzicń po" -jestna rcceptę; której E dostaniesz, trafisz nalekarza zasłaniającego się klauzą sumicnia Możcsz się tcz zdc rzyć z sumieniem aptekarskim choćto niezgodnezprawem wte dynic uda cisiękupić żadncj antykonccpcyjncj ochrony. W dużym miescie to nie kłopot, zarogicm jest inna aptcka. W miasteczkualbona wsiniera Dlatego dziś spotykamy się W stu miejscowościach (godzing imiejscespotkań w swoim miescie zajdziccic m.in.na wysokicob nie 'jeśli wyboru casy pl). Właśnie przede wszyst kim wtej sprawie mozliwości wyboru Bo wbrcw opiniom na ukowcow 'sprzed wiekówinaszej politycznej wierchuszki, niejesteśmy bezmyślnymi króliczkami. Iwzięcicnas wkarby tonicnajlepszy sposób napoprawędzietości Cała Europajuz wic, boto sprawdzila, żedziecisięrodzą kiedyich mamymają zagwarantowaną pracęizlobck Ipcwność, że państwo im pomoże; gdy trafiim się nieprzyjemnymąż czy partner: Tymczasem prezydent naszego kraju, kwestionując przcgłosowaną przcz poprzedni parlament konwencję antyprzemocowa; pyta: po coto przyjmujcmy Itojcst po szóstc. Po siódme zamiast calowania wrączkę; a władzęmamy gorliwie szarmancką chccmy równych płac. Po ósme prawdziwego sza cunku, który polcga na uznaniu naszego prawa do prywatności; intymnościi decydowania sobie Awkońcu po dziewiąte-nie uychodździsbez parasolki, boma padać 