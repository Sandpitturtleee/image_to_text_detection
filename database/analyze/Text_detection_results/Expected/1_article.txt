Dziś pierwsze urodziny tego symbolu kobiecej siły i siostrzeństwa ponad podziałami - ubiegłego 3 października powstrzymałyśmy rząd przed wprowadzeniem całkowitego zakazu aborcji. Ale władza nie śpi - to rząd nocnych jarków i by nie dać się zaskoczyć, nie możemy parasolek składać. Po pierwsze dlatego że chcą nam zabrać in vitro. Po drugie: środowisko antykobiece, którym PiS sprzyja, wciąż zbierają podpisy pod projektami ustaw całkowicie zakazującymi aborcji i antykoncepcji. Równocześnie - i to po trzecie - dobra zmiana w szkole edukację seksualną zastąpiła katechizmem. Po czwarte - utrudnia dostęp do antykoncepcji. Ta awaryjna - tabletka "dzień po" - jest na receptę, której nie dostaniesz, jeśli trafisz na lekarza zasłaniającego się klauzulą sumienia. Możesz też się zderzyć z sumieniem aptekarskim - choć to niezgodne z prawem - wtedy nie  uda ci się kupić żadnej antykoncepcyjnej ochrony. W dużym mieście to nie kłopot, za rogiem jest inna apteka. W miasteczku albo na wsi nie ma wyboru. Dlatego dziś spotykamy się w stu miejscowościach (godzinę i miejsce spotkań w swoim mieście znajdziecie miń. Na wysokieobcasy.pl). Właśnie przede wszystkim w tej sprawie - możliwość wyboru. Bo wbrew opiniom naukowców sprzed wieków i naszej politycznej wierchuszki, nie jesteśmy bezmyślnymi króliczkami. I wzięcie nas w karby to nie najlepszy sposób na poprawę dzietności. Cała Europa już wie, bo to sprawdziła, że dzieci się rodzą, kiedy ich mamy mają zagwarantowaną pracę i żłobek. I pewność że państwo im pomoże, gdy trafi im się nieprzyjemny maż czy partner. Tymczasem prezydent naszego kraju, kwestionując przegłosowaną przez poprzedni parlament konwencję antyprzemocową, pyta: po co to przyjmujemy. I to jest po szóste. Po siódme - zamiast całowania w rączkę, a mamy władzę gorliwie szarmancką - chcemy równych płac. Po ósme - prawdziwego szacunku, który polega na uznaniu naszego prawa do prywatności, intymności i decydowania o sobie. A w końcu po dziewiąte - nie wychodź dziś bez parasolki, bo ma padać.